Lesson4/src/Homeworck4_1/Computer.java

```diff
@@ -0,0 +1,62 @@
+package Homeworck4_1;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Computer {
+    private String processor;
+    private int operational;
+    private int drive;
+    private int counter;
+    private int condition = 1;
+
+    public Computer(String processor, int operational, int drive) {
+        this.processor = processor;
+        this.operational = operational;
+        this.drive = drive;
+
+    }
+
+
+    public void Info() {
+        System.out.println("processor\t" + processor + "\noperational\t" + operational + "\ndrive\t" + drive + "\ncounter\n" + counter);
+    }
+
+    public void Enable() {
+        this.counter++;
+        if (this.condition == 0 || this.counter > 3) {
```

Comment:
counter limit (3) should be passed from console

Reply:
this check should happen right after 43 line because a computer can burn on turning on according to the task description